docs/users-documentation.html

```diff
@@ -164,16 +152,39 @@
                         <ol>
                             <li>Clone the repository: git clone https://github.com/project-ncl/dependency-analysis</li>
                             <li>Enter the directory with the CLI: cd dependency-analysis/cli-wrap/src/main/resources/cli/</li>
-                            <li>Change permissions: chmod +x *.py *.sh</li>
-                            <li>Run the CLI. E.g.: ./da-cli.sh lookup org.hibernate:hibernate-core:4.2.21.Final</li>
+                            <li>Change permissions: chmod +x *.py</li>
+                            
                         </ol>
                 </p>
             </li>
+            
+            <li>
+                After unpacking/clonning CLI tool, you need to configure servers.
+            <p>
+                You have to specify DA server and KeyCloak authentication server. It can be done by inserting URL addresses of these servers to file config.json 
+                DA server should be filled into "daServer" field. This server has to support WebSocket and HTTP protocol. Keycloak server should be in field "keycloakServer".
+                There are also "keycloakUser" and "keycloakRealm" fields, which should be filled by your credentials for proper working scripts. 
```

Comment:
keycloakUser field does not exist. Also it's not users credentials, it's ... well let's say keycloak configuration.